src/styles/search.css

```diff
@@ -0,0 +1,81 @@
+.search {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin: 20px auto 0;
+  height: var(--large_input_height);
+}
+
+.search--small {
+  height: var(--small_input_height);
+}
+
+.search__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search__label::after {
+  content: "";
+  display: block;
+  background-image: url(../images/Search.svg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  position: absolute;
+  width: var(--large_icon);
+  height: var(--large_icon);
+  top: 25px;
+  left: 26px;
+}
+
+.search__label--small::after {
+  width: var(--small_icon);
+  height: var(--small_icon);
+  top: 15px;
+  left: 13px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  font-family: inherit;
+  color: #3d4e61;
+
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search__input--small {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.search__input:hover {
+  box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
```

Comment:
You can move repeating values to variables

Reply:
hey! thank you for your advices and review